### PR TITLE
Add Scripture Component

### LINKED ISSUE #216: Port over Scripture Component from NS Apollos
## User Story

As a developer I want to be able to create scripture components so that I can provide users with a better content experience.

#### Problem Details _(Optional)_

* Screenshots, GIFs, or a video
* Any additional information (version, device, etc.)

#### Acceptance Criteria
* You can see scripture using this component in Storybook.

### PR DESCRIPTION
**Pull request by richarddubay**
Fixes #216 

## Creator

- [x] Build relevant tests if any apply
- [x] Ensure there are no new warnings (in app AND in storybook)
- [x] Upload an animated GIF showcasing the solution (if it applies)
- [x] Set a relevant reviewer
- [x] PR has a relevant title that adheres to our patch-notes/change-log style

## Reviewer

- [ ] Review CI build results
- [ ] Review function and form on iOS
- [ ] Review function and form on Android
- [ ] Review new stories if any apply
- [ ] Review test coverage: Are all new or changed states tested?
- [ ] Review new test logic
- [ ] Review the change-log entry
- [ ] Review that the PR title adheres to our patch-notes/change-log style

### CODE REVIEW

**Comment by richarddubay** [comment]
screenshot 2018-08-27 12 03 14

File: packages/apolloschurchapp/src/ui/Paragraph.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { compose } from 'recompose';
+import { View } from 'react-native';
+
+import {
+  withPlaceholder,
+  Paragraph as ParagraphPlaceholder,
+} from 'apolloschurchapp/src/ui/Placeholder';
+import styled from 'apolloschurchapp/src/ui/styled';
+
+const Paragraph = compose(
```
Comment (conrad-vanl) [inline packages/apolloschurchapp/src/ui/Paragraph.js]: This component was already ported over here - https://github.com/ApollosProject/apollos-prototype/tree/master/packages/apolloschurchapp/src/ui/typography/Paragraph

Comment (richarddubay) [reply packages/apolloschurchapp/src/ui/Paragraph.js]: That's a great catch. I didn't even look in that folder for that component!

**Comment by richarddubay** [comment]
@conrad-vanl and @burkeshartsis This is feeling pretty good. Would love y'all to take a look and let me know if anything else needs to be done here.

**Comment by richarddubay** [comment]
@conrad-vanl I approve this message. We definitely need this to get merged (so I'm going to) so that we can continue working on the rest of the scripture/devotional stuff.

**Pull request merged**